freqtrade/arguments.py

```diff
@@ -456,7 +454,21 @@ def testdata_dl_options(self) -> None:
             type=str,
             metavar='PATH',
         )
-
+        self.parser.add_argument(
+            '-d', '--datadir',
```

Comment:
At a later point, we should try to combine as many options as possible (--datadir is used 4 times in this file, and i guess it's similar with many other options) (however not for this PR ...)

Reply:
yeah, it should go somehow to a common part of the options (but not for all commands though). Further improvement.